readme.md

```diff
@@ -182,23 +182,35 @@ input values, and does not produce side effects.
 const greet = (name) => 'Hi, ' + name
 
 greet('Brianne') // 'Hi, Brianne'
-
 ```
 
-As opposed to:
+As opposed to each of the following:
 
 ```js
-
-let greeting
+window.name = 'Brianne'
 
 const greet = () => {
-  greeting = 'Hi, ' + window.name
+  return 'Hi, ' + window.name
```

Comment:
Also, if we're going to update this code to use interpolation instead of concatenation, I'll update the *pure* example above it to do the same. I think it's most clear if all three code examples follow the same pattern as much as possible while maintaining clarity.